src/parser/parser.py

```diff
@@ -67,5 +68,6 @@ def _parse_items(self):
 
 
 if __name__ == '__main__':
-    parser = Parser()
+    parser = Parser(language='spanish')
+    #parser = Parser(language='english')
```

Comment:
Keep english as our default

Reply:
Can ignore this part; its temporary for testing purposes. It will be properly parameterized later, likely by another module.

Reply:
The default in the class definition is english currently